docs/src/styles/fonts.scss

```diff
@@ -0,0 +1,39 @@
+@font-face {
+  font-family: 'Amazon Ember';
+  src: url('/fonts/AmazonEmber_W_Rg.woff2') format('woff2');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
```

Comment:
I don't think anything in the global nav uses the 500 weight; just 400 and 700. And probably you can drop the italics font files as well.